### DISCUSSION TITLE
Fotos

### DISCUSSION THREAD

**Discussion by Rtanxc** [Ryanlucass/ChaveMestra#2] [Ideas]
As foto das telas poderia ficar armazenadas no readme apontando o link para o projeto do figma de contribuição

**Comment by Ryanlucass** [comment]
Realmente vou fazer isso, fica mais fácil a organização

**Comment by Ryanlucass** [comment]
> As foto das telas poderia ficar armazenadas no readme apontando o link para o projeto do figma de contribuição
Realmente vou fazer isso, fica mais fácil a organização
